Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/AssetAttribute/Asset.tsx

```diff
@@ -124,20 +125,32 @@ export namespace AssetAttributes {
                         else
                             props.UpdateState(AssetAttributes.getNewAsset('Line'));
                     }}
-                    //Disabled={props.NewEdit == 'Edit'}
+                    Disabled={!props.AllowEdit ?? true}
```

Comment:
This nullish coalescing operator will never do anything. Javascript reads the negation first, and thus assumes undefined means false. Same result in this case, but if we wanted to change default behavior, someone may miss this detail and wonder why changing true to false doesn't do anything.
```suggestion
                    Disabled={!(props.AllowEdit ?? true)}
```